pints/_transformation.py

```diff
@@ -0,0 +1,574 @@
+#
+# Transformation functions
+#
+# This file is part of PINTS (https://github.com/pints-team/pints/) which is
+# released under the BSD 3-clause license. See accompanying LICENSE.md for
+# copyright notice and full license details.
+#
+from __future__ import absolute_import, division
+from __future__ import print_function, unicode_literals
+import pints
+import numpy as np
+from scipy.special import logit, expit
+
+
+class Transform(object):
+    """
+    Abstract base class for objects that provide some convenience parameter
+    transformation from the model parameter space to a search space.
+
+    If ``t`` is an instance of a ``Transform`` class, you can apply
```

Comment:
Maybe not `t` but e.g. `trans` ?
(Also maybe `q` instead of `p`? Or are we using `$\theta$` everywhere?)

Reply:
Use `trans`. But keep `p` for model parameter and `q` as transformed search space parameter (adapting your comment below).